internal/ipmi/ipmi.go

```diff
@@ -383,3 +383,40 @@ func (i *Ipmi) ClearSystemEventLog(ctx context.Context) (err error) {
 	_, err = i.run(ctx, []string{"sel", "clear"})
 	return err
 }
+
+// GetSystemEventLog returns the system event log entries in ID, Timestamp, Description, Message format
+func (i *Ipmi) GetSystemEventLog(ctx context.Context) (entries [][]string, err error) {
+	output, err := i.GetSystemEventLogRaw(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting system event log")
+	}
+
+	scanner := bufio.NewScanner(strings.NewReader(output))
+	for scanner.Scan() {
+		line := strings.Split(scanner.Text(), "|")
+		if len(line) < 3 {
+			continue
+		}
+		if line[0] == "ID" {
+			continue
+		}
+		for i := range line {
+			line[i] = strings.TrimSpace(line[i])
+		}
+
+		// ID, Timestamp (date time), Description, Message (message : assertion)
```

Comment:
can you include some fixture data for this, along with a test for the parsing (move this scanner block into another method if that makes it easier to test)